frontend.py

```diff
@@ -27,18 +29,22 @@
 sites = ' '.join(list(patterns['username_patterns'].keys()))
 logos = json.dumps(patterns['logos'])
 signup = ' '.join(list(patterns['signup'].keys()))
+real_name = 'no'
```

Comment:
I am not sure, but wouldn't using booleans would be better over here?

Reply:
How?

The username is not available at that point of the program. So there's nothing to check against.

Reply:
And the bool isn't parsed by js correctly as python uses CamelCase.